client/src/game/reducers/initialStates/initialState.ts

```diff
@@ -32,6 +32,22 @@ export default function initialState(metadata: GameMetadata): State {
     datetimeStarted: null,
     datetimeFinished: null,
 
+    // because of play stack notes (e.g. throw it in a hole)
```

Comment:
use full sentences in notes in a capital letter and no ending punctuation, e.g. 
// The array needs to be longer than just the total cards in the deck because we also need to account for notes on a stack base